src/pages/SellProducts.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import SellProducts from '../../pages/SellProducts';
+
+it('should render active listings', () => {
+    render(<SellProducts />);
+
+    expect(screen.getAllByRole('heading')[0]).toHaveTextContent(
+        'Sell Products'
+    );
+
+    // active listings panel should be enabled on render
+    expect(screen.getByTestId('active-listings-btn')).toBeEnabled();
+
+    // TODO: mock active listings call and expect render accordingly
```

Comment:
Is this done?

Reply:
This will be done in a following PR with actual API call. Right now its just all dummy data